save-preprocessor/src/main/kotlin/com/saveourtool/save/preprocessor/controllers/DownloadProjectController.kt

```diff
@@ -543,8 +541,7 @@ class DownloadProjectController(
         }
             .collectList()
             .flatMap {
-                testSuiteIds.sort()
-                execution.testSuiteIds = testSuiteIds.joinToString()
+                execution.formatAndSetTestSuiteIds(testSuiteIds)
```

Comment:
now we collect testSuiteIds to some mutable list and after all requests we update execution.
Suggestion: move update testSuiteIds to step when we create tests on backend (append to existed testSuiteIds)

Reply:
will be done in next PR